packages/card/README.md

```diff
@@ -0,0 +1,35 @@
+# Archetype
+
+Mineral UI `Card` component. Find out more about [Mineral UI](https://github.com/mineral-ui/mineral-ui).
+
+
+## Installation
+
+1. Install project [peer dependencies](../../docs/peer-dependencies.md), if not already installed
+
+2. Install the package
+
+  ```sh
+  npm install --save @mineral-ui/card
+  ```
+
+
+## Usage
+
+```jsx
+import React from 'react';
+import { Card, CardTitle, CardBlock } from '@mineral-ui/card';
```

Comment:
> I am suggesting this because everything is still coming from the@mineral-ui/card package and it makes my brain think that the default would be a Card. I think this would be a good pattern for us to follow.

Agreed.